src/main/java/eu/bbmri/eric/csit/service/negotiator/database/model/Post.java

```diff
@@ -46,5 +46,8 @@ public class Post extends AuditEntity {
   private Resource resource;
 
   @Lob private String text;
+
   private String status;
```

Comment:
If you have type as Enum then maybe we can do the same for Status?

Reply:
Yes, That's a good idea. I'll fix it

Reply:
Fixed in 5cb22c0